render-wasm/src/wasm/text.rs

```diff
@@ -368,7 +368,7 @@ pub extern "C" fn update_shape_text_layout_for_all() {
 
 #[no_mangle]
 pub extern "C" fn get_caret_position_at(x: f32, y: f32) -> i32 {
-    with_current_shape!(state, |shape: &Shape| {
+    with_state_mut_current_shape!(state, |shape: &Shape| {
```

Comment:
can't we use here directly "with_current_shape"? I think there's no need of `with_state_mut_current_shape` as far as I know, at least not for the current changes, right?